openedx/core/djangoapps/programs/tests/factories.py

```diff
@@ -65,3 +65,15 @@ class Meta(object):
     completed = []
     in_progress = []
     not_started = []
+
+
+class ProgramCredential(factory.Factory):
```

Comment:
This is really a credential factory. Create a factories module in credentials/tests/.